### ISSUE TITLE
All your Captcha problems explained - One issue to answer all 50 

### ISSUE THREAD

**Issue by V4NSH4J** [V4NSH4J/discord-mass-DM-GO#488]
**What Changed?**
Discord implemented captchas on [joining guilds](https://i.imgur.com/sV2aeEr.png) (If they think your account might be involved in spam) and on a certain number of [direct messages](https://gyazo.com/ea237c707930d773b425b41cc9c18c2b).

**So what's the problem? What's this new kind of captcha that you can't use public solver APIs?**
The captcha coming on joining servers and sending DMs is an invisible HCaptcha. I know you can see it, but in the network requests, it behaves as if it were an invisible captcha. The new captchas come with [extra fields](https://i.imgur.com/zyOlnZ2.png) `rqdata` and `rqtoken`.  This is not exactly a problem, as some captcha solver APIs like https://capmonster.cloud https://2captcha.com and https://rucaptcha.com do accept `rqdata` field for their captchas. `rqtoken` is sent back to discord along with the solved captcha key. This is perhaps a step to have a unique solution for each captcha unlike before. Many services like antigate (anti-captcha), azcaptcha, etc do not support this new field which is why they can't be used any longer. And for the ones which do work, you can include all necessary information while passing the hcaptcha but my guess is the bot score for these captchas is high so Discord flags the accounts in result you get reduced number of DMs and are not able to join servers. And in most scenerios, you'd get the captcha again. 

**Why can't you use Hcaptcha Accessibility cookies?** 
Using an Hcaptcha accessibility cookie on joining server will lock your token and considering it's the same captcha now for sending DMs, it will soon be like that as well my guess is. 

**So what can you do?**
You can wait. This most likely would require you to have solvers in the future unless we're somehow able to get low bot scores from the public APIs. As of writing this, DMDGO v1.8.11 is the latest version and it completely bypasses the captcha on DMs but it will mostly be patched soon after which captcha APIs will be the only interim solution. Instead of spamming me regarding hcaptchas, if you have any ideas, feel free to share them here or make a pull request and implement them. 

**What is Error 25 while using proxy_for_captcha**
The captcha service does not like your proxy. Please read their docs for what proxies they accept and contact their support if you have any queries. 

If your issue was closed linking this, feel free to ask your questions below (it was done as a quick way of answering everyone needing help), I'll keep this open till it's active or for the next few days.

**Comment by itschasa** [comment]
the amount of references lmao

**Comment by DevastMod** [comment]
nice work on understanding how it works, so if i use paid 2captcha solvers api n stuff it still wont work? cause i dont get captchas on joining but when DMing most often

**Comment by V4NSH4J** [comment]
> nice work on understanding how it works, so if i use paid 2captcha solvers api n stuff it still wont work? cause i dont get captchas on joining but when DMing most often

Right, it would work but most times discord will reject the captcha and ask you to solve it again. And for the times it does work, your token will be locked in a few solves.

**Comment by king3435** [comment]
bypass patched again,can it be solved again this time anytime soon?

**Comment by V4NSH4J** [comment]
> bypass patched again,can it be solved again this time anytime soon?

Let's see, still pretty early to say. If I find a solution it will be implemented.

**Comment by V4NSH4J** [comment]
Update: Capmonster.cloud has been fixed and is working well at the moment.

**Comment by V4NSH4J** [comment]
Update: Capmonster.cloud is patched, anti-captcha has added support for Enterprise Hcaptcha just today and it's working pretty well.

**Comment by ChippuArt** [comment]
What kind of proxy service you d recommend for anti-captcha? I use rainproxy, but I get the error 24 somehting is wrong with the proxy type. @V4NSH4J 🙇‍♂️🙇‍♂️🙇‍♂️😎Thanks for any help I will read their docs maybe I will get smarter ....

**Comment by Alisha8141l** [comment]
how to use anti captcha ?

**Comment by Storaqe** [comment]
so this still is a problem?

**Comment by BadBadBoss** [comment]
Is capmonster still patched?

**Comment by TrustGod001** [comment]
Can anyone recommend a site to buy a good quality token.
